src/communication/communication/position_subscriber.py

```diff
@@ -0,0 +1,69 @@
+"""
+Node that subscribes to localization topics and sends data over HTTP as MQTT Topic.
+"""
+import json
+import rclpy
+from rclpy.node import Node
+import requests
+from interfaces.msg import Position
+
+class PositionSubscriber(Node):
```

Comment:
Try testing it with a random node that sends the same localization data to your node